tobac/tests/test_tracking.py

```diff
@@ -557,3 +559,94 @@ def test_untracked_nat():
     # the exact data type depends on architecture, so
     # instead just check by name
     assert output["time_cell"].dtype.name == "timedelta64[ns]"
+
+
+@pytest.mark.parametrize(
+    "cell_time_lengths, min_time_length, expected_there",
+    [
+        (
+            [0, 5, 10, 120, 300],
+            0,
+            [True, True, True, True, True],
+        ),
+        (
+            [0, 5, 10, 120, 300],
+            5,
+            [False, False, True, True, True],
+        ),
+        (
+            [0, 5, 10, 120, 300],
+            6,
+            [False, False, True, True, True],
+        ),
+        (
+            [0, 5, 10, 120, 300],
+            120,
+            [False, False, False, False, True],
+        ),
+        (
+            [0, 5, 10, 120, 300],
+            900,
+            [False, False, False, False, False],
+        ),
+    ],
+)
+def test_time_cell_min(
+    cell_time_lengths: list[int],
+    min_time_length: int,
+    expected_there: list[bool],
+):
+    """
+    Tests time_cell_min in particle-based tracking
+    Parameters
+    ----------
+    cell_time_lengths: list[int]
+        Length that each cell appears for
+    min_time_length: int
+        time_cell_min value
+    expected_there: list[bool]
+        whether to expect the cell there.
+
+    """
+    # delta time automatically set by smallest difference between cell lengths
+    delta_time = int(np.min(np.diff(cell_time_lengths)))
```

Comment:
Why go through the struggle of this line and have to use floor division twice, when you could just set `num_frames` to the original cell time lengths (so that `delta_time` would be 1, no division needed) and then assert against the original cell time lengths again at the end?

Reply:
Good point. I've updated the code just to set `delta_time` to 1.